contrib/IECoreUSD/test/IECoreUSD/data/sphere.usda

```diff
@@ -0,0 +1,15 @@
+#usda 1.0
+(
+  defaultPrim = "hello"
+)
+
+
+def Xform "hello"
+{
+    def Sphere "world"
+    {
+        float3[] extent = [(-2, -2, -2), (2, 2, 2)]
+        color3f[] primvars:displayColor = [(0, 0, 1)]
+        double radius = 2
+    }
+}
```

Comment:
I can't see this file being used anywhere - was it perhaps included by mistake?

Reply:
Originally there was only a reading test and I referenced this file. I have another PR which directly follows this one which uses the file but will remove it from here.